tests/unit/test_zowe_core.py

```diff
@@ -380,6 +388,49 @@ def test_profile_not_found_warning(self, get_pass_func):
             prof_manager = ProfileManager()
             prof_manager.config_dir = self.custom_dir
             props: dict = prof_manager.load("non_existent_profile")
+    
+    @patch("keyring.get_password", side_effect=keyring_get_password)
+    def test_profile_loading_with_valid_schema(self, get_pass_func):
+        """
+        Test Validation, no error should be raised for valid schema
+        """
+        # Setup - copy profile to fake filesystem created by pyfakefs
+        custom_file_path = os.path.join(self.custom_dir, "zowe.config.json")
+        shutil.copy(self.original_nested_file_path, custom_file_path)
+        shutil.copy(self.original_schema_file_path, self.custom_dir)
+        os.chdir(self.custom_dir)
+
+        self.setUpCreds(custom_file_path, {
+            "profiles.zosmf.properties.user": "user",
+            "profiles.zosmf.properties.password": "password",
+        })
+
+        # Test
+        prof_manager = ProfileManager(appname="zowe")
+        prof_manager.config_dir = self.custom_dir
+        props: dict = prof_manager.load(profile_name="zosmf")
+    
+    @patch("keyring.get_password", side_effect=keyring_get_password)
+    def test_profile_loading_with_invalid_schema(self, get_pass_func):
+        """
+        Test Validation, no error should be raised for valid schema
+        """
+        # Setup - copy profile to fake filesystem created by pyfakefs
+        with self.assertRaises(ValidationError):
+            custom_file_path = os.path.join(self.custom_dir, "invalid.zowe.config.json")
```

Comment:
> Note: You may want to be cautious when selecting which files to copy, because the test can get confusing very quickly 😋

Should we generate invalid JSON files programmatically to place in the fake file system? For example, we could load a valid JSON file from disk, change one of the properties to make it invalid, and then save the invalid JSON to the fake filesystem. I think this would reduce the number of JSON files needed in the fixtures folder and hopefully make the tests a bit less confusing 🙂